### PR TITLE
Added cronjobs to the performance chapter

### PR DESCRIPTION
**Pull request by Toflar**

### CODE REVIEW

File: docs/manual/performance/cronjobs.de.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: 'Cronjob Framework'
+description: 'Das Contao Cronjob Framework einrichten.'
+url: "performance/cronjobs"
+aliases:
+    - /de/performance/cronjobs/
+---
+
+Contao wird standardmässig mit einem Cronjob-Framework ausgeliefert. Kurz zusammengefasst ermöglicht dies
```
Comment (netzarbeiter) [inline docs/manual/performance/cronjobs.de.md]:
```suggestion
Contao wird von Haus aus mit einem Cronjob-Framework ausgeliefert. Kurz zusammengefasst ermöglicht dies
```

File: docs/manual/performance/cronjobs.en.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: 'Cronjob Framework'
+description: 'Setting up the Contao Cronjob framework.'
+aliases:
+    - /en/performance/cronjobs/
+---
+
+Contao ships with a cronjob framework by default. In short, this allows developers to register cronjobs
```
Comment (netzarbeiter) [inline docs/manual/performance/cronjobs.en.md]:
```suggestion
Contao ships with a cronjob framework by design. In short, this allows developers to register cronjobs 
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Yanick.

File: docs/manual/performance/cronjobs.en.md (hunk 2 above)
Comment (Toflar) [reply docs/manual/performance/cronjobs.en.md]: `default` is better than `design` in my opinion :)

Comment (netzarbeiter) [reply docs/manual/performance/cronjobs.en.md]: Two times by default.

Comment (Toflar) [reply docs/manual/performance/cronjobs.en.md]: Changed.

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**